eogrow/utils/batch.py

```diff
@@ -6,7 +6,7 @@
 
 def read_timestamps_from_orbits(userdata: dict) -> List[str]:
     """Parses batch orbits payload to obtain a list of timestamp strings"""
-    if "orbits" in userdata:
+    if "orbits" in userdata:  # noqa SIM908
```

Comment:
> [SIM908](https://github.com/MartinThoma/flake8-simplify/issues/50): Use dict.get(key) ([example](https://pypi.org/project/flake8-simplify/#SIM908))

what a stupid flake8 simplification, why not ignore it all-together?

Reply:
because in most cases it is better to use `get`. The issue is how we're doing the whole *you are you, unless you're a dictionary in which case you are your subitem*, which is just bad practice.

Reply:
Why not then fix the problem at the source? I'd be happy to lose a bad practice habit. Or is this case an exception?

Reply:
I'd say this is an exception (for now) due to how the timestamps reader is done